src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -71,12 +71,12 @@ export class GuUnhealthyInstancesAlarm extends GuAlarm {
       ...props,
       alarmName: alarmName,
       alarmDescription: alarmDescription,
-      metric: props.targetGroup.metricUnhealthyHostCount(),
+      metric: props.targetGroup
+        .metricUnhealthyHostCount()
+        .with({ period: Duration.minutes(5), statistic: Statistic.MAXIMUM }),
```

Comment:
This resolves a deprecation warning where `period` and `statistic` should now be declared against the `metric`, rather than a top level prop.